server/real_estate/src/main/java/com/realestate/real_estate/repos/Offer.java

```diff
@@ -0,0 +1,29 @@
+package com.realestate.real_estate.repos;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Builder
+@Jacksonized
+@Document(collection = "offers")
+public class Offer {
+    @Id
+    ObjectId id;
```

Comment:
Let's change this to `private String id`
I ran into a problem trying to delete from our database using `ObjectId`